config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_logginglogmetrics.logging.cnrm.cloud.google.com.yaml

```diff
@@ -402,19 +390,10 @@ spec:
               updateTime:
                 description: Output only. The last update timestamp of the metric.
                   This field may not be present for older metrics.
-                format: date-time
                 type: string
             type: object
-        required:
-        - spec
         type: object
     served: true
     storage: true
     subresources:
       status: {}
-status:
```

Comment:
FYI this change is good, it's a bug in controller-tools that we output the CRD status fields.